classes/analytics/analytics_for_recordings_table.php

```diff
@@ -165,5 +174,33 @@ public function get_sort_columns() {
         return [$this->sort_default_column => $this->sort_default_order];
     }
 
-}
+    public function build_table() {
+
```

Comment:
Kill this empty line

Reply:
This was a nasty core gross syntax. I was aiming to leave it 1:1 with the tablelib.php implementation as much as I could, but I agree its yucky.